frontend-public/package.json

```diff
@@ -17,7 +17,6 @@
     "filepond-plugin-file-validate-type": "^1.2.1",
     "filepond-polyfill": "^1.0.2",
     "hoist-non-react-statics": "^3.2.1",
-    "js-file-download": "^0.4.4",
```

Comment:
Neither of your package-lock.json files appear to have been updated. This will cause problems with `npm ci`. If you changed the package.json directly, you'll need to run `npm install` to get the package-lock in sync

Reply:
Thanks 👍